audit_exceptions/permitted_formula_license_mismatches.json

```diff
@@ -1,5 +1,2 @@
 [
-  "cmockery",
-  "vapoursynth-ocr",
-  "vapoursynth-sub"
```

Comment:
GitHub and formulae seem to match now (`MIT`). Didn't see any issue with `brew audit --online ...`